### PR TITLE
Added tests for T1098.004.

### PR DESCRIPTION
**Pull request by haresudhan**
**Details:**
Added tests for T1098.004.

**Testing:**
tested locally on CentOS and MacOS.

**Associated Issues:**
None

### CODE REVIEW

File: atomics/T1098.004/T1098.004.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+atomic_tests:
+- name: Modify SSH Authorized Keys
+  description: |
+    Modify contents of <user-home>/.ssh/authorized_keys to maintain persistence on victim host. 
+    If the user is able to save the same contents in the authorized_keys file, it shows user can modify the file.
+  supported_platforms:
+    - macos
+    - linux
+  executor:
+    name: sh
```
Comment (rc-grey) [inline atomics/T1098.004/T1098.004.yaml:14]: Id be explicit here and call out `bash` as not all shells support the `$()` operator.

**Review by rc-grey — CHANGES_REQUESTED** [review]
Looks good, one minor change for distros that don't have an bourne based shell.

**Comment by haresudhan** [comment]
Thanks @rc-grey. Changed to bash executor.

File: atomics/T1098.004/T1098.004.yaml (hunk 1 above)
Comment (haresudhan) [reply atomics/T1098.004/T1098.004.yaml]: Thanks @rc-grey. Changed to bash executor.

**Review by rc-grey — APPROVED** [review]

**Pull request merged**